src/Tribe/Views/V2/View.php

```diff
@@ -1393,7 +1393,16 @@ protected function setup_template_vars() {
 			? Dates::build_date_object( $event_date )->format( Dates::DBDATEFORMAT )
 			: false;
 
-		$template_vars = [
+		/*
+		 * Some plugins, like WooCommerce, will modify the UID of logged out users; avoid that filtering here.
+		 *
+		 * @see TEC-3579
+		 */
+		$rest_nonce = tribe_without_filters( [ 'nonce_user_logged_out' ], static function () {
+			return wp_create_nonce( 'wp_rest' );
+		} );
+
+		$template_vars   = [
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 3 spaces

<sub>hash: 2cae0f95f1223b4d001dcaf9a7fdde0f</sub>

Reply:
```suggestion
		$template_vars = [
```